test/test_wait.py

```diff
@@ -72,8 +74,9 @@ def test_wait_for_socket(wfs, spair):
     assert wfs(a, read=True, timeout=0)
 
     # Waiting for a socket that's actually been closed is just a bug, and
-    # raises an appropriate exception:
-    with pytest.raises(ValueError):
+    # raises some kind of helpful exception (exact details depend on the
+    # platform).
+    with pytest.raises(Exception):
```

Comment:
Could this `Exception` be made more specific so we're not masking other failures?

Reply:
Probably, but it seems to depend on both python version and OS, and the operation we're testing there is total nonsense that can't possibly work, doesn't add any coverage, and should never happen in correct code, so it doesn't seem worth trying to curate a list of which exact exceptions happen in which configurations?

Reply:
But what I've seen so far is `ValueError` and `socket.error`, which could become the initial list of you want.

Reply:
That is true, you've convinced me.